core/package.json

```diff
@@ -26,7 +26,6 @@
     "vue-carousel": "^0.18",
     "vue-i18n": "^8.0.0",
     "vue-meta": "^2.3.3",
-    "vue-observe-visibility": "^0.4.1",
```

Comment:
`vue-observe-visibility"` is in 2 package.json files and you removed it only in the one, why?